### PR TITLE
feat: added first driver example

### PR DESCRIPTION
**Pull request by alexfournierahizoune**
### Type
- [X] New feature
- [ ] Refactoring
- [ ] Bug-fix
- [ ] DevOps
- [ ] Testing

### Description
Added driver code example as part of [HOR-852](https://linear.app/connektica/issue/HOR-852/[doc]-add-complex-examples-for-worker-task-doc)
[Source](https://github.com/QCoherence/python_drivers/blob/master/ZNB20.py)

### Additional Notes
This documentation will be referenced by the worker-task internal/public documentation.

### CODE REVIEW

File: examples/drivers/ZNB20.py [hunk 1]
```diff
@@ -0,0 +1,1373 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+
+from instrument import Instrument
+import qt
```
Comment (jmdeschenes) [inline examples/drivers/ZNB20.py]: Unless I misunderstand, qt is one heck of a big library to import. Why do we need this?

File: examples/drivers/ZNB20.py [hunk 2]
```diff
@@ -0,0 +1,1373 @@
...
+    reset = True|False)
+    '''
+
+    def __init__(self, name, address, reset = False):
+        '''
+        Initializes the ZNB20
+
+        Input:
+            name (string)    : name of the instrument
+            address (string) : TCPIP/GPIB address
```
Comment (jmdeschenes) [inline examples/drivers/ZNB20.py]: An example of how that string should look like would be great.

File: examples/drivers/ZNB20.py [hunk 3]
```diff
@@ -0,0 +1,1373 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+
+from instrument import Instrument
```
Comment (jmdeschenes) [inline examples/drivers/ZNB20.py]: how does that work? What do you need installed to make this work?

File: examples/drivers/ZNB20.py [hunk 4]
```diff
@@ -0,0 +1,1373 @@
...
+import qt
+import visa
+import logging
```
Comment (jmdeschenes) [inline examples/drivers/ZNB20.py]: imports order:

builtin

external packages

inner packages

```
import types
import logging

import numpy as np
from numpy import pi
```

File: examples/drivers/ZNB20.py [hunk 5]
```diff
@@ -0,0 +1,1373 @@
...
+
+            ####################################################################
+            # we have to look at the errors checking of freq start and freq stop....
+            ####################################################################
+
+            asked_freq_start = self._visainstrument.query('SEGM%s:FREQ:STAR?'% segment_number)
+            given_freq_start = str(startfrequency*1e9)
+
+            if np.float(asked_freq_start) != np.float(given_freq_start):
+                print'error in setting start frequency at %s' %startfrequency
```
Comment (jmdeschenes) [inline examples/drivers/ZNB20.py]: How does that work? Is this a python snippet?

File: examples/drivers/ZNB20.py [hunk 6]
```diff
@@ -0,0 +1,1373 @@
...
+
+        # We check the type of traces elements
+        if not all([type(trace) is str for trace in traces]):
+            raise ValueError('Element in traces should be string type')
+
+        logging.info(__name__ +\
+                     ' : start to measure and wait till it is finished')
+
+        while self._visainstrument.query('*ESR?') != '1':
+            qt.msleep(0.1)
```
Comment (jmdeschenes) [inline examples/drivers/ZNB20.py]: Only place where we use the package. Can we replace it with `time.sleep`?

**Review by jmdeschenes — CHANGES_REQUESTED** [review]
This example if for python 2
How did you test this?

**Comment by alexfournierahizoune** [comment]
@jmdeschenes this code example is not and will not be used anywhere, it's only to showcase how to port it to our "framework". It's linked in the worker tasks documentation. I only grabbed the code from a github repo to use as reference.

**Review by jmdeschenes — APPROVED** [review]

**Pull request merged**